From: Sven-Thorsten Dietrich <sdietrich@suse.de>
Subject: Update smp_send_nmi_allbutself  mask parameter

Patch patches.arch/x86_sgi_cpus4096-05-update-send_IPI_mask.patch
changes the mask parameter for send_IPI_mask to a pointer.

Update the RT version.

Signed-off-by: Sven-Thorsten Dietrich <sdietrich@suse.de>
---
 arch/x86/kernel/nmi.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/arch/x86/kernel/nmi.c
+++ b/arch/x86/kernel/nmi.c
@@ -591,6 +591,6 @@ void smp_send_nmi_allbutself(void)
 	cpumask_t mask = cpu_online_map;
 	cpu_clear(safe_smp_processor_id(), mask);
 	if (!cpus_empty(mask))
-		send_IPI_mask(mask, NMI_VECTOR);
+		send_IPI_mask(&mask, NMI_VECTOR);
 #endif
 }
